packages/cli/test/commands/migrate/config-ts.ember-app.test.ts

```diff
@@ -0,0 +1,165 @@
+import { resolve } from 'node:path';
+import { readdirSync } from 'node:fs';
+import { afterEach, beforeEach, afterAll, describe, expect, test, vi } from 'vitest';
+import { readJSONSync, writeJSONSync } from 'fs-extra/esm';
+import {
+  getEmberAppProject,
+  getEmberAppWithInRepoAddonProject,
+  getEmberAppWithInRepoEngineProject,
+} from '@rehearsal/test-support';
+import { Project } from 'fixturify-project';
+import { tsConfigTask } from '../../../src/commands/migrate/tasks/index.js';
+import { listrTaskRunner, createMigrateOptions } from '../../test-helpers/index.js';
+import { runTsConfig, createUserConfig } from '../../test-helpers/config-ts-test-utils.js';
+import { TSConfig } from '../../../src/types.js';
+import { UserConfig } from '../../../src/user-config.js';
+
+const projects = {
+  emberApp: getEmberAppProject(),
+  emberAppWithInRepoAddon: getEmberAppWithInRepoAddonProject(),
+  emberAppwithInRepoEngine: getEmberAppWithInRepoEngineProject(),
+};
+
+describe('Task: config-ts ember app', () => {
+  for (const [name, originalProject] of Object.entries(projects)) {
+    describe(name, () => {
+      let project: Project;
+      let output = '';
+
+      beforeEach(async () => {
+        project = originalProject.clone();
+        await project.write();
+        output = '';
+
+        vi.spyOn(console, 'info').mockImplementation((chunk: string) => {
+          chunk = chunk.replace(new RegExp(project.baseDir, 'g'), '<tmp-path>');
+          output += `${chunk}\n`;
+        });
+
+        vi.spyOn(console, 'log').mockImplementation((chunk) => {
+          output += `${chunk}\n`;
+        });
+      });
+
+      afterEach(() => {
+        vi.clearAllMocks();
+        output = '';
+        project.dispose();
+      });
+
+      afterAll(() => {
+        originalProject.dispose();
+      });
+
+      test('create tsconfig if not existed', async () => {
+        await runTsConfig(project.baseDir);
+
+        expect(readJSONSync(resolve(project.baseDir, 'tsconfig.json'))).matchSnapshot();
+        expect(output).matchSnapshot();
+      });
+
+      test('update tsconfig if exists', async () => {
+        // Prepare old tsconfig
+        const oldTsConfig = { compilerOptions: { strict: false } };
+        writeJSONSync(resolve(project.baseDir, 'tsconfig.json'), oldTsConfig);
+
+        await runTsConfig(project.baseDir);
+
+        const tsConfig = readJSONSync(resolve(project.baseDir, 'tsconfig.json')) as TSConfig;
+
+        expect(tsConfig.compilerOptions.strict).toBeTruthy();
+        // Do not use snapshot here since there is absolute path in output
```

Comment:
see my previous comment on dynamic paths. you'll need to sanitize the path from the input like we are doing in tests so the snapshot contains eg https://github.com/rehearsal-js/rehearsal-js/blob/f89124d5050bbf787e1524602f1ab97831ff51f6/packages/cli/test/commands/migrate/__snapshots__/convert.test.ts.snap#L40

Reply:
... same for all comments in this PR around `// Do not use snapshot here since there is absolute path in output`

Reply:
fixed